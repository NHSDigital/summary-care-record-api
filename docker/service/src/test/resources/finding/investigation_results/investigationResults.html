<QUPC_IN210000UK04>
    <ControlActEvent>
        <subject>
            <GPSummary>
                <pertinentInformation2 typeCode="PERT" inversionInd="false" contextConductionInd="true" negationInd="false">
                    <templateId root="2.16.840.1.113883.2.1.3.2.4.18.2" extension="CSAB_RM-NPfITUK10.pertinentInformation1" />
                    <seperatableInd value="true" />
                    <pertinentCREType classCode="CATEGORY" moodCode="EVN">
                        <code code="163141000000104" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="Investigation Results" />
                        <component typeCode="COMP" contextConductionInd="true">
                            <templateId root="2.16.840.1.113883.2.1.3.2.4.18.2" extension="CSAB_RM-NPfITUK10.component" />
                            <seperatableInd value="false" />
                            <UKCT_MT144043UK02.Finding classCode="OBS" moodCode="EVN">
                                <id root="3CDB62E6-EFEA-4036-9709-52AE7ED7D930" />
                                <code code="1322781000000102" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="Severe acute respiratory syndrome coronavirus 2 antigen detection result positive (finding)">
                                </code>
                                <statusCode code="active" />
                                <effectiveTime>
                                    <low value="20200805" />
                                </effectiveTime>
                            </UKCT_MT144043UK02.Finding>
                        </component>
                    </pertinentCREType>
                </pertinentInformation2>
            </GPSummary>
        </subject>
    </ControlActEvent>
</QUPC_IN210000UK04>
