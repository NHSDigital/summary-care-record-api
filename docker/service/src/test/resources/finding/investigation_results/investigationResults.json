{
  "resourceType": "Observation",
  "id": "3CDB62E6-EFEA-4036-9709-52AE7ED7D930",
  "meta": {
    "profile": [ "https://fhir.hl7.org.uk/StructureDefinition/UKCore-Observation" ]
  },
  "identifier": [ {
    "value": "3CDB62E6-EFEA-4036-9709-52AE7ED7D930"
  } ],
  "status": "final",
  "category": [ {
    "coding": [ {
      "system": "http://snomed.info/sct",
      "code": "163141000000104",
      "display": "Investigation Results"
    } ]
  } ],
  "code": {
    "coding": [ {
      "system": "http://snomed.info/sct",
      "code": "1322781000000102",
      "display": "Severe acute respiratory syndrome coronavirus 2 antigen detection result positive (finding)"
    } ]
  },
  "effectivePeriod": {
    "start": "2020-08-05"
  }
}
