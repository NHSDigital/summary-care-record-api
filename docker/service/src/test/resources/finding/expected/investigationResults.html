<pertinentInformation2 typeCode="PERT" inversionInd="false" contextConductionInd="true" negationInd="false">
    <templateId root="2.16.840.1.113883.2.1.3.2.4.18.2" extension="CSAB_RM-NPfITUK10.pertinentInformation1"/>
    <seperatableInd value="true"/>
    <pertinentCREType classCode="CATEGORY" moodCode="EVN">
        <code code="163141000000104" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="Investigation Results"/>
        <component typeCode="COMP" contextConductionInd="true">
            <templateId root="2.16.840.1.113883.2.1.3.2.4.18.2" extension="CSAB_RM-NPfITUK10.component"/>
            <seperatableInd value="false"/>
            <UKCT_MT144043UK02.Finding classCode="OBS" moodCode="EVN">
                <id root="19ABC6D1-8AF5-11EA-9FCE-AFDCAECF9DFB"/>
                <code code="163131000000108" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) RNA (ribonucleic acid) detection result positive"/>
                <statusCode code="completed"/>
                <effectiveTime>
                    <low value="20200805"/>
                </effectiveTime>
            </UKCT_MT144043UK02.Finding>
        </component>
    </pertinentCREType>
</pertinentInformation2>
